docs/configuration.md

```diff
@@ -1506,6 +1506,11 @@ Apart from these, the following properties are also available, and may be useful
     of this setting is to act as a safety-net to prevent runaway uncancellable tasks from rendering
     an executor unusable.
   </td>
+  <td><code>spark.stage.maxConsecutiveAttempts</code></td>
+  <td>4</td>
+  <td>
+    Number of consecutive stage retries allowed before a stage is aborted.
```

Comment:
Hah sorry for all of the comment changes from the combination of Imran and me!! But I agree that this was an issue before and would be good to update.  Thanks for the many updates here @sitalkedia.